lib/mayaHydra/mayaPlugin/renderOverride.h

```diff
@@ -125,6 +125,9 @@ class MtohRenderOverride : public MHWRender::MRenderOverride
     MStatus setup(const MString& destination) override;
     MStatus cleanup() override;
 
+    // Utility function to get HdStorm GPU memory usage stats
+    static int GetHdStGPUMemory();
```

Comment:
probably better to rename it as GetUsedGPUMemory, assuming _renderDelegate->GetRenderStats() is renderer agnostic